doc/oidc/auth0.md

```diff
@@ -0,0 +1,211 @@
+
+``` text
+09:02:54.527 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Starting...
+09:02:54.737 [main] INFO  com.zaxxer.hikari.pool.HikariPool - HikariPool-1 - Added connection conn0: url=jdbc:h2:mem:lrsql.h2 user=
+09:02:54.739 [main] INFO  com.zaxxer.hikari.HikariDataSource - HikariPool-1 - Start completed.
+09:02:54.740 [main] INFO  lrsql.system.database - Starting new connection for h2:mem database...
+09:02:54.807 [main] INFO  lrsql.system.lrs - Starting new LRS
+09:02:54.829 [main] WARN  lrsql.util.cert - No cert files found. Creating self-signed cert!
+09:02:56.106 [main] INFO  org.eclipse.jetty.util.log - Logging initialized @7146ms to org.eclipse.jetty.util.log.Slf4jLog
+09:02:56.323 [main] INFO  org.eclipse.jetty.server.Server - jetty-9.4.48.v20220622; built: 2022-06-21T20:42:25.880Z; git: 6b67c5719d1f4371b33655ff2d047d24e171e49a; jvm 11.0.17+8-LTS
+09:02:56.353 [main] INFO  o.e.j.server.handler.ContextHandler - Started o.e.j.s.ServletContextHandler@75ad8ca8{/,null,AVAILABLE}
+09:02:56.374 [main] INFO  o.e.jetty.server.AbstractConnector - Started ServerConnector@3e9f3d0f{HTTP/1.1, (http/1.1, h2c)}{0.0.0.0:8080}
+09:02:56.378 [main] INFO  o.e.jetty.util.ssl.SslContextFactory - x509=X509@16e693c1(lrsql_keystore,h=[com.yetanalytics.lrsql],a=[],w=[]) for Server@5192abb4[provider=null,keyStore=null,trustStore=null]
+09:02:56.412 [main] INFO  o.e.jetty.server.AbstractConnector - Started ServerConnector@3387d45e{SSL, (ssl, alpn, h2, http/1.1)}{0.0.0.0:8443}
+09:02:56.413 [main] INFO  org.eclipse.jetty.server.Server - Started @7453ms
+09:02:56.413 [main] INFO  lrsql.system.webserver - Starting new webserver at host 0.0.0.0, HTTP port 8080, and SSL port 8443
+09:02:56.413 [main] INFO  lrsql.system.webserver -
+   __  ______     _   _
+|\ \ \ \     |   | | | |     _        _               _       _
+| \ \_\ \    |   | |_| |    | |_    / _ \            | |_   _| |_ _
+| /   __ \   |    \   / ____| __|  |  _  |  _    ____| | |_| |  _|_|  ___   ___
+|/ /\ \ \ \  |     | | | / /| |    | | | |/ _ \ / /_ | |\   /| |  _ / __/  / __|
+  /  \ \ \ \ |     | | |  /_| |_   | | | | | | | |_| | | | | | |_| | |___  _| |
+ /____\ \_\ \|     |_| |____|\__|  |_| |_|_| |_|\__|_|_| |_|  \__|_|\____/___/
+        _______    _______    _            _          _______    _______
+       |  _____|  |  ___  |  | |          | |        |  ___  |  |  _____|
+       | |_____   | |   | |  | |          | |        | |___| |  | |_____
+       |_____  |  | |   | |  | |          | |        |   ____|  |_____  |
+        _____| |  | |__ | |  | |_____     | |_____   | |\  \     _____| |
+       |_______|  |_____\_\  |_______|    |_______|  |_|  \__\  |_______|
+                        (c) 2021-2022 Yet Analytics Inc.
+
+```
+
+#### Log in via Auth0
+
+Navigate to [http://localhost:8080](http://localhost:8080) and click "OIDC LOGIN":
+
+![OIDC login button](../images/oidc/auth0/20_oidc_login_0.png)
```

Comment:
...we're going to want to fix the CSS on that page at some point.

Reply:
yeah it's a bit odd